extending.html

```diff
@@ -2224,15 +2224,16 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
                     same feature twice.
                     Refer to Karaf forum posting at http://karaf.922171.n3.nabble.com/Duplicate-feature-repository-entry-using-archive-kar-to-build-deployable-applications-td3650850.html
                     --&gt;</span>
+                    <span class="tag">&lt;resourcesDir&gt;</span>${project.build.directory}/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
 
                     <span class="comment">&lt;!--
-                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.2, it will need to be
+                    Location of the features.xml file. If it references properties that need to be filtered, e.g., ${project.version}, it will need to be
```

Comment:
Should ${project.version} and ${project.build.finalName} render to the actual version?